proposals/extensions.md

```diff
@@ -424,39 +420,37 @@ static class CollectionExtensions
     }
     extension<T>(HashSet<T>)
     {
-        public static HashSet<T> Create() { ... }
+        public static HashSet<T> Create() { ... } // error
     }
 }
 ```
 
 ## Open issues
 
+- Confirm `extension` vs. `extensions` as the keyword
+
 ### Metadata
 
-- Should we emit all implementation methods with speakable names instead, as a disambiguation strategy and also to allow
-  usage from other languages? We could add an attribute to handle compile-time conflicts in factory scenario (`[ExtensionName("CreateList")]`).
-- The metadata format currently doesn't include any modreqs to block other compilers. But the spec does mention we
-  would block those scenarios. Let's either remove this requirement or update the metadata format.  
-- We should follow-up on "factory scenario" where multiple extension declarations have static factory methods 
-  with same parameter types but different return types.
-- Should the extension marker or unspeakable implementation methods be marked with special name?
```

Comment:
Should the question about special name remain? I didn't see changes related to that in other places and I do not remember LDM making decision on this.